.github/workflows/ci.yml

```diff
@@ -42,7 +42,7 @@ jobs:
           - {VERSION: "3.11", NOXSESSION: "tests", OPENSSL: {TYPE: "libressl", VERSION: "3.7.3"}}
           - {VERSION: "3.11", NOXSESSION: "tests", OPENSSL: {TYPE: "libressl", VERSION: "3.8.0"}}
           - {VERSION: "3.11", NOXSESSION: "tests-randomorder"}
-          - {VERSION: "3.12-dev", NOXSESSION: "tests"}
+          - {VERSION: "3.12", NOXSESSION: "tests"}
```

Comment:
Why is this preferable?

Reply:
Some people prefer to include `allow-prereleases: true` so that they can omit the `-dev`, meaning there's no need to tidy it up later. But not much difference otherwise.